client/spec.yaml

```
@@ -495,4 +495,178 @@ paths:
           description: Bad request
         "5XX":
           description: Server side error while tailing logs
-
+  /api/logs/v1/{tenant}/loki/api/v1/rules:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+    get:
+      tags:
+        - logs/rulesv1
+      summary: Get Logs Rules for a tenant in JSON form
+      operationId: getAllLogsRules
+      description: |
+        You can get Logs rules for a tenant with health data
+      responses:
+        "2XX":
+          description: rule for a tenant
```

```suggestion
          description: All log rules for a tenant
```
I think we could change the description to something on that direction, since `getAllLogsRules` returns more than one rule (I think the same suggestion applies to the other endpoints that return more than one rule)